omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -724,9 +731,40 @@
                     jq().toastmessage('showErrorToast', " Patient Not Stable " + instabilityCause + ". \n Only Stable Patients can be enrolled in " + jq("#165143").find("select").find(":selected").text());
                     return false;
                 }
+
+                if (getValue('art-regimen.value') != "" &amp;&amp; getValue('no-of-art-pills.value') == "") {
+                     getField('no-of-art-pills.error').html('No. of pills required').show();
+                    return false;
+                }
+
+                if (getValue('isoniazid-trigger.value') != "" &amp;&amp; getValue('isoniazid-pills.value') == "") {
+                    getField('isoniazid-pills.error').html('No. of pills required').show();
+                    return false;
+                }
+
+                if (getValue('b6-trigger.value') != "" &amp;&amp; getValue('b6-pills.value') == "") {
+                    getField('b6-pills.error').html('No. of pills required').show();
+                    return false;
+                }
+
+                if (getValue('99413-target.value') != "" &amp;&amp; getValue('160856.value') == "") {
+                    getField('160856.error').html('No. of pills required').show();
+                    return false;
+                }
+
+                if (getValue('fluconazole-trigger.value') != "" &amp;&amp; getValue('fluconazole-pills.value') == "") {
+                    getField('fluconazole-pills.error').html('No. of pills required').show();
+                    return false;
+                }
+
+                if (getValue('other-meds-trigger.value') != "" &amp;&amp; getValue('other-meds-quantity.value') == "") {
+                    getField('other-meds-quantity.error').html('Quantity required').show();
```

Comment:
@dbaluku message 'Quantity of other medications is required' - can you get the name of the selected medication to make this more user friendly

Reply:
Done